bin.js

```diff
@@ -262,6 +262,8 @@ const result = commist()
   })
   .register('doctor', catchify(async function (argv) {
     const version = require('@nearform/doctor/package.json').version
+    checkArgs(argv, 'clinic-doctor', version);
```

Comment:
No semi-colon unless the code complains. I believe we're using `standard` for style linting here.

Reply:
@jasnell - Oh dear. I didn't realise I put them in. I've always put semi colons at the end of lines in JS. This will be a habit I need to change. 👍

Reply:
Fixed listing issues in latest commit.